### ISSUE TITLE
Unable to delete a compose

### ISSUE THREAD

**Issue by ralphbean** [product-definition-center/product-definition-center#369]
I'm trying to delete a compose that was imported improperly (and then I'll re-import it aftewards).

I tried:

```bash
curl -X DELETE -H "Content-Type: application/json" https://pdc.fedoraproject.org/rest_api/v1/composes/299/  -H "Authorization: Token $TOKEN"
```

but I get back
```json
{"detail":"Not found."}
```

So... something is not found, but it is not clear *what* is not found.  Is it the compose?  That definitely exists.  Is it some association?  Which one?

**Comment by ralphbean** [comment]
(FYI, I just manually deleted that compose in the db so we could re-import it).

**Comment by simozhan** [comment]
@ralphbean 299 is not a compose_id, you need to delete the compose with the compose_id
URL: /rest_api/v1/composes/{compose_id}/

**Comment by lubomir** [comment]
Just a minor addition: even if you use correct identifier, the delete call in API does not actually delete anything, it just marks the compose as deleted. Importing it again will probably not work the way you expect.

**Comment by ralphbean** [comment]
Oh!  Silly me.

I think I got confused about the compose ID because:

- I started here:  https://pdc.fedoraproject.org/compose/
- Then clicked on a compose and got to here:  https://pdc.fedoraproject.org/compose/83/

And then I pulled the ``id`` from the url (when I should've looked for the ``compose_id``.

Maybe that non-api view could use the ``compose_id`` instead of the ``id`` just to keep expectations consistent for users?  That's kind of off-topic for this ticket though.  Closing as 'not a bug'.  Thanks!
